web/src/components/TaskStatusChangeSnackbar.tsx

```diff
@@ -10,8 +10,10 @@ interface Props {
 export const TaskStatusChangeSnackbar = (props: Props): ReactElement => {
   const { Config } = useConfig();
   return (
-    <SnackbarAlert variant="success" isOpen={props.isOpen} close={props.close}>
-      {Config.taskDefaults.taskProgressSuccessSnackbarBody}
-    </SnackbarAlert>
+    <>
+      <SnackbarAlert variant="success" isOpen={props.isOpen} close={props.close}>
+        {Config.taskDefaults.taskProgressSuccessSnackbarBody}
+      </SnackbarAlert>
+    </>
```

Comment:
[Pebble] This doesn't need to be wrapped in a fragment

Reply:
Yes! I forgot to delete it when I removed the other component invocation. TY!